merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/ResultsProtocol.java

```diff
@@ -35,6 +38,11 @@ public interface WriterRole {
     //   of the underlying resource in order to deallocate it.
     void succeedWith(SimulationResults results);
     void failWith(String reason);
+    default void failWith(final Throwable throwable) {
```

Comment:
A newline between the abstract methods and the default (helper) method might be nice. 😃

Reply:
Newline added 🖖